flexget/plugins/daemon/scheduler.py

```diff
@@ -108,7 +108,7 @@ def setup_scheduler(manager):
         timezone = tzlocal.get_localzone()
         if timezone.zone == 'local':
             timezone = None
-    except pytz.UnknownTimeZoneError:
+    except:
```

Comment:
Too wide exception catch, this is not good python. It will catch everything under the sun including aborting program with ctrl-c. What exception did you encounter? Add it into the list instead. eg

    except (pytz.UnknownTimeZoneError, SomethingElse):